src/main/kotlin/uk/gov/justice/digital/hmpps/personrecord/service/CourtCaseEventsService.kt

```diff
@@ -45,7 +45,7 @@ class CourtCaseEventsService(
         } else if (matchesExistingRecordPartially(defendants, person)) {
           log.info("Partially matching CPR record exists for defendant - no further processing will occur")
           telemetryService.trackEvent(TelemetryEventType.NEW_CASE_PARTIAL_MATCH, extractMatchingFields(defendants[0], person))
-        } else {
+        } else if (defendants.isEmpty()) {
```

Comment:
I think this will prevent the retry of any of the search failures, no?

Reply:
Agree, but I still believe we should be retrying any of the search failures so will discuss that with them.